charts/numaflow/templates/deployment.yaml

```diff
@@ -238,7 +261,8 @@ spec:
             name: numaflow-server-rbac-config
           name: rbac-config
 
-### numaflow server validating webhook
+### numaflow server validating webhook, deploy only when numaflow is installed in cluster scope
```

Comment:
I think we should give a standalone config to the validating webhook, install or not install. It could also be installed when the controller is installed in namespace scope.